rest-api/dao/physical_measurements_dao.py

```diff
@@ -281,6 +281,7 @@ def _update_participant_summary(self, session, obj):
     participant_summary.physicalMeasurementsFinalizedTime = obj.finalized
     participant_summary.physicalMeasurementsCreatedSiteId = obj.createdSiteId
     participant_summary.physicalMeasurementsFinalizedSiteId = obj.finalizedSiteId
+    participant_summary.lastModified = clock.CLOCK.now()
```

Comment:
Add a check for this here:

https://github.com/all-of-us/raw-data-repository/blob/master/rest-api/test/unit_test/dao_test/physical_measurements_dao_test.py#L96

Reply:
done